.github/workflows/ci.yml

```diff
@@ -15,24 +15,24 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8']
-        toxenv: [quality, docs, 'py38']
+        python-version: ['3.9']
+        toxenv: [quality, docs, 'py39']
```

Comment:
We should upgrade the Python version to 3.11 as this is version currently used in most IDAs